honeybee_radiance_command/rfluxmtx.py

```diff
@@ -0,0 +1,139 @@
+"""rfluxmtx command"""
+
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+from .options.rfluxmtx import RfluxmtxOptions, RfluxmtxControlParameters
+
+
+# octree to octree and so on
+class Rfluxmtx(Command):
+    """
+    Rfluxmtx command.
+
+    Rfluxmtx computes the flux transfer matrices for a RADIANCE scene.
+
+    """
+
+    __slots__ = ('_input', "_sensors", "_sender", "_octree", "_receivers", "_system")
+
+    def __init__(self, options=None, output=None, sensors=None, sender=None, receivers=None, system=None,
+                 octree=None):
+        """Initialize Command."""
+        Command.__init__(self, output=output)
+        self.options = options
+        self.sensors = sensors
+        self.octree = octree
+        self.sender = sender
+        self.receivers = receivers
+        self.system = system
+
+    @property
+    def options(self):
+        """Rfluxmtx options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = RfluxmtxOptions()
+
+        if not isinstance(value, RfluxmtxOptions):
+            raise ValueError('Expected Rfluxmtx options not {}'.format(value))
+
+        self._options = value
+
+    @property
+    def sensors(self):
+        """Sensor file."""
+        return self._sensors
+
+    @sensors.setter
+    def sensors(self, value):
+        if value is None:
+            self._sensors = None
+        else:
+            self._sensors = typing.normpath(value)
+
+    @property
+    def sender(self):
+        """Sender file."""
+        return self._sender
+
+    @sender.setter
+    def sender(self, value):
+        if value is None:
+            self._sender = None
+        else:
+            self._sender = typing.normpath(value)
+
+    @property
+    def receivers(self):
+        """Receivers file."""
+        return self._receivers
+
+    @receivers.setter
+    def receivers(self, value):
+        if value is None:
+            self._receivers = None
+        else:
+            self._receivers = typing.normpath(value)
+
+    @property
+    def system(self):
+        """System file. Note that rfluxmtx can accept any number of system files, however, to keep
+        the implementation clean, only one system file is being allowed. (SS: 23.Sep.2021)"""
```

Comment:
We don't use dated comments in docstring. Also, we try to do it the PEP8 way. Which is one short line followed by an explanation line if necessary.

```suggestion
        """System file.
        
        Note that rfluxmtx can accept any number of system files, however, to keep
        the implementation clean, only one system file is being allowed.
        """
```